CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v4.3.0 (Unreleased)
```

Comment:
Since 4.2.1 is not released yet, it is just the new _possible_ version of the SDK, instead creating a new entry (4.3.0) just because you have a new resource it is not necessary, you can simply rename 4.2.1 to 4.3.0 and add the new resource headers there, otherwise you'll need two separate development branches, one for each version.

Reply:
4.3.0 was just released, this should likely be bumped to `4.4.0 (Unreleased)`.